archinstall/lib/interactions/disk_conf.py

```diff
@@ -175,9 +175,9 @@ def select_disk_config(
 	return None
 
 
-def _boot_partition(sector_size: disk.SectorSize) -> disk.PartitionModification:
+def _boot_partition(sector_size: disk.SectorSize, using_gpt: bool) -> disk.PartitionModification:
```

Comment:
If people call this through their own scripts, this would break those. Could we add a default `=None` here, and check if `is None` and log [RequirementError](https://github.com/archlinux/archinstall/blob/f6106324eb71905df9b832eb973a08a1f45fa5af/archinstall/lib/exceptions.py#L7)?

Reply:
The function should not be used externally, it starts with an underscore to indicate it is intended to be private.

Reply:
I do agree with that point, qe shouldn't concern ourselves with people calling internal functions when they shouldn't

Reply:
Fair point.